### PR TITLE
Update untracked repos and clean up metadata

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Brings in all the repos we weren't already tracking from ansible-collections - there's a new metadat key that tracks if the repo is included in the package or not.

### CODE REVIEW

File: config/collections.yml [hunk 1]
```diff
@@ -2,448 +2,529 @@ default:
...
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.molecule
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
```
Comment (gundalow) [inline config/collections.yml:149]: community.molecule is not in the `ansible` package

File: config/collections.yml [hunk 2]
```diff
@@ -2,448 +2,529 @@ default:
...
-    NewCollection: 'FALSE'
-    MergeKey: vmware
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: vmware_rest
-    Regex: vmware
-    NewCollection: 'FALSE'
-    MergeKey: vmware
+    in_package: yes
```
Comment (gundalow) [inline config/collections.yml:197]: `vmware_rest` is not currently in `ansible` package, though should be for ansible 5.0.0

File: config/collections.yml [hunk 3]
```diff
@@ -2,448 +2,529 @@ default:
...
-    NewCollection: 'FALSE'
-    MergeKey: windows
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.yang
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
```
Comment (gundalow) [inline config/collections.yml:205]: `community.yang` is not in `ansible` package

**Review by gundalow — COMMENTED** [review]
Thanks, just a couple of minor corrections.

File: config/collections.yml (hunk 2 above)
Comment (GregSutcliffe) [reply config/collections.yml]: Thanks I'll set it to no for now then, we can change it later

**Comment by GregSutcliffe** [comment]
Thanks @gundalow changed all of those. will push now, we can always iterate more

**Pull request merged**